site/quickstart.md

```diff
@@ -0,0 +1,70 @@
+* Create a PV & PVC domain; use script
+* Configure the operator to manage the domains in the domain namespace using `helm upgrade`
+* Configure the Traefik load balancer to manage the domains in the domain namespace.
+  * Use `helm upgrade` to configure the load balancer to monitor the Ingresses in the domain namespace
+* Create the domain home
+* Create the Docker image for the domain home in the image or use the WebLogic binary image.
+  * Run WLST to create the domain in PV (remember to apply patch).
+
+## 6. Create a domain.
+* Edit the domain YAML file (can the defaults be used?)
+* Create the domain home for the domain
+  * For a domain home on PV – as in sample
+  * For a domain home in image, use the sample in the Docker GitHub project
+* Optionally, create a configuration overrides template and any additional Kubernetes secrets it needs (for example, to override the domain home configuration of a database URL, username, and password).
+* Create a domain resource in the domain namespace
+  * Specify the following information: domain UID, service account, secret name, the domain home details, and optionally, the configuration overrides template name.
+* Configure the operator to know about the domain
+* Configure the Traefik load balancer to manage the domain as follows:
+  * Create an Ingress for the domain in the domain namespace (it contains the routing rules for the domain)
+
+(At this point, do they have a WebLogic Kubernetes deployment in a Kubernetes cluster? If so, we have to give them something to look at, to verify their results.)
+
+(Removing the domain, operator, and such, isn't really needed in a quick start guide. We can point them to the user's instructions for how to remove the domain, operator, and all the other related resources.)
+
+## 7. Remove a domain.
+* Remove the domain's Kubernetes resources (domain, secrets, ingress, ...)
+  * If they are all tagged with a `weblogicUID` label, then the sample delete domain script will remove them all  
+  * The operator will notice that the domain's domain resource has been removed and will then kill the pods
+* Configure the Traefik load balancer to stop managing the domain
+  * The delete script will have deleted the Ingress which will inform the load balancer to stop managing the domain
+* Remove the domain home if it's on a PV.
+
+## 8. Remove the domain namespace.
+* Configure the Traefik load balancer to stop managing the domain namespace. Use `helm upgrade` to remove the domain namespace from the list of namespaces.
+* Configure the operator to stop managing the domain. Use `helm upgrade` to remove the domain namespace from the list of domain namespaces
+* Remove the PV & PVC for the domain namespace
+* Remove the domain namespace
+
+## 9. Remove the operator.
+* `helm delete --purge`
+* Remove the operator namespace
+
+## 10. Remove other resources.
+* Optionally, remove Kibana and Elasticsearch
```

Comment:
To remove them, use:
$   kubectl apply -f kubernetes/samples/scripts/elasticsearch_and_kibana.yaml

Reply:
Got it.